common/OpTestHMC.py

```diff
@@ -863,6 +861,35 @@ def is_msp_enabled(self, mg_system, vios_name, remote_hmc=None):
         if int(msp_output[0]) != 1:
             return False
         return True
+    
+    def is_perfcollection_enabled(self):
+        '''
+        Get Performance Information collection allowed in hmc profile
+
+        :returns: Ture if  allow_perf_collection in hmc otherwise false
+        '''
+
+        rc = self.run_command("lssyscfg -m %s -r lpar --filter lpar_names=%s -F allow_perf_collection"
+                                % (self.mg_system, self.lpar_name))
+        if  rc:
+            return True
+        
+        return False
+
+    def hmc_perfcollect_configure(self, enable=True):
+        '''
+        Enable/Disable perfcollection from HMC
+        enable perfcollection value is 1 and for disable it 0
+        '''
+        # Set perf collection profile  value using HMC command
+        cmd = ('chsyscfg -r lpar -m %s -i "name=%s, allow_perf_collection=' %
+               (self.mg_system, self.lpar_name))
+        if enable:
+            cmd = '%s1"' % cmd
+        else:
+            cmd = '%s0"' % cmd
```

Comment:
if check better to have it before  the cmd 
and also better to have a variable to allow_perf_collection=perf_collect
```
perf_collect = 0
if enable:
    perf_collect = 1

cmd = ('chsyscfg -r lpar -m %s -i "name=%s, allow_perf_collection=%s' %
               (self.mg_system, self.lpar_name, perf_collect))
```

Reply:
optimised code lines

Reply:
i don't feel it will optimize as using extra variable increase garbage collector job :)